app/src/main/kotlin/com/wire/android/ui/common/SurfaceBackgroundWrapper.kt

```diff
@@ -9,7 +9,7 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 
 @Composable
-fun WhiteBackgroundWrapper(content: @Composable () -> Unit, modifier: Modifier) {
+fun SurfaceBackgroundWrapper(content: @Composable () -> Unit, modifier: Modifier = Modifier) {
```

Comment:
what if make it: 
`fun SurfaceBackgroundWrapper(modifier: Modifier = Modifier, content: @Composable () -> Unit)` ? 
i guess in that case we could call it like: 
`SurfaceBackgroundWrapper(modifier = Modifier) { /* content is here */}` so it make code a bit cleaner IMO

Reply:
true, going to submit it in a sec